rosetta/cosmos/launchpad/network.go

```diff
@@ -61,6 +63,80 @@ func (l Launchpad) NetworkOptions(ctx context.Context, request *types.NetworkReq
 	}, nil
 }
 
+type latestBlockResponse struct {
+	Block block `json:"block"`
+}
+
+type genesisResponse struct {
+	Result genesisResult `json:"result"`
+}
+
+type genesisResult struct {
+	Block block `json:"block"`
+}
+
+type netInfoResponse struct {
+	Result netInfoResult `json:"result"`
+}
+
+type netInfoResult struct {
+	Peers []peer `json:"peers"`
+}
+
 func (l Launchpad) NetworkStatus(ctx context.Context, request *types.NetworkRequest) (*types.NetworkStatusResponse, *types.Error) {
-	panic("implement me")
+	var (
+		latestBlockResp   latestBlockResponse
+		genesistBlockResp genesisResponse
+		netInfoResp       netInfoResponse
+	)
+
+	g, ctx := errgroup.WithContext(ctx)
+	g.Go(func() error {
+		resp, err := l.request(ctx, l.cosmos(endpointBlockLatest), nil)
+		if err != nil {
+			return err
+		}
+		defer resp.Body.Close()
+		return json.NewDecoder(resp.Body).Decode(&latestBlockResp)
+	})
+	g.Go(func() error {
+		resp, err := l.request(ctx, l.tendermint(endpointNetInfo), nil)
+		if err != nil {
+			return err
+		}
+		defer resp.Body.Close()
+		return json.NewDecoder(resp.Body).Decode(&netInfoResp)
+	})
+	g.Go(func() error {
+		resp, err := l.request(ctx, l.tendermint(endpointBlock), nil)
```

Comment:
I think you need to tell to get the block 1, now it returns by default the latest, not the genesis.

Reply:
This is important before closing the PR.

Reply:
solved by 2a3e660